integration-test-groups/foundation/bean/src/test/java/org/apache/camel/quarkus/component/bean/BeanTest.java

```diff
@@ -246,9 +246,8 @@ public void notReducedTest() {
         RestAssured.given()
                 .get("/bean/allBeanInstances")
                 .then()
-                .body("size()", is(5))
+                .body("size()", is(4))
                 .body(allOf(
-                        containsString("defaultBean"),
```

Comment:
Not sure why previously we had this behaviour. When fetching all beans for a type it would return a default bean, even if it had been overridden. Which seems wrong to me.

With this change, that wont happen. Hopefully it does not break anything.

Reply:
Probably a left over, let's see the test suite.